gem/admin.py

```diff
@@ -57,6 +62,58 @@ def gender(self, obj):
         return obj.gem_profile.get_gender_display()
 
 
+class GemFrontendUsersResource(FrontendUsersResource):
+        gender = Field()
+
+        class Meta:
+            export_order = FrontendUsersResource.Meta.export_order + (
+                'gender',)
+
+        def dehydrate_gender(self, user):
+            if hasattr(user, 'gem_profile'):
+                return user.gem_profile.get_gender_display()
+            return None
+
+
+class GemFrontendUsersAdminView(IndexView):
```

Comment:
@justinvdm is there a way to override this post's dataset? Tried a few things but I could just be making simple errors